src/evals/sequence_completion_capability.py

```diff
@@ -5,17 +5,14 @@
 from tqdm.auto import tqdm
 
 from src.models.completions import generate_response_with_turns
-from src.pipelines.sequence_completions import (
-    PromptType,
-    generate_sequence_completion_prompt,
-)
+from src.pipelines import ShotSamplingType, TaskType
+from src.pipelines.sequence_completions import generate_sequence_completion_prompt
 from src.utils import auto_subdir
```

Comment:
@domenicrosati how is this used? Please check the function below and the data path. Perhaps use similar function from `hydra.utils`?